.gitignore

```diff
@@ -5,6 +5,7 @@
 
 # logs
 *.log
+**/core
```

Comment:
The wildcard is too broad? Will it affect `cpp/core`?

Reply:
> The wildcard is too broad? 
Thats as expected since we wouldn't ever want a core file to be added to git as they can be upwards of a GB. 
> Will it affect cpp/core?
If you mean the folder, then no. To effect folders the regex needs to have a `/` at the end.

Reply:
@zhztheplayer Updated it to end at core. Thanks for catching.